spec/support/factory_girl.rb

```diff
@@ -16,8 +16,9 @@
   end
 
   factory :review do
-    sequence(:header) { |n| "review#{n}" }
-    body "This is the body of the review. It's terribly interesting."
+    sequence(:header) { |n| "review #{n}" }
+    sequence(:body) { |n| "#{n}. This is the body of the review.
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.